inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -148,6 +238,19 @@ class XmlSerializer : public ngraph::AttributeVisitor {
         m_xml_node.append_attribute(name.c_str())
             .set_value(create_atribute_list(adapter).c_str());
     }
+    void on_adapter(
+        const std::string& name,
+        ngraph::ValueAccessor<std::shared_ptr<Function>>& adapter) override {
+        pugi::xml_document xml_body;
```

Comment:
Can we use this `on_adapter()` in order to serialize all functions?

Reply:
It is already used in order to serialize nGraph function to fill body part for tensor iterator.

Reply:
I believe that we can use it inside the Serialization pass.

Reply:
on_adapter() is implemented in Serialization pass.